### ISSUE TITLE
Decide on learning rate decay schedule.

### ISSUE THREAD

**Issue by amj** [tensorflow/minigo#25]
We've used `tf.train.exponential_decay(1e-2, global_step, 10 ** 7, 0.1)`.  This isn't quite right, and it's possible it's the source of some of our overfitting woes.  Let's check our math to get an equivalent to the AGZ paper.

**Comment by killerducky** [comment]
LZ's policy is to wait until progress stalls, only then lower. Since LZ has regular promotion matches there is an easy way to define stalling. Minigo could keep the always promote policy but add some matches as a way to measure this. Or use CGOS results.

**Comment by amj** [comment]
Learning rate of .1/.01/.001 for 200k/400k/800k steps (per AGZ, 2048 batch) equates to  400M/800M/1.6B positions

**Comment by amj** [comment]
(for my later reference) On training batch size of 32, that's 12.8M steps / 25.6M steps / 51.2 M steps.

**Comment by amj** [comment]
We still don't really know the right answer here.  We're using the rates in paper 2 but it seems to be based more off of the l2_cost.
